CC/Sounder/recorder.cc

```diff
@@ -63,7 +63,7 @@ Recorder::Recorder(Config* in_cfg, unsigned int core_start)
     } catch (std::exception& e) {
         std::cout << e.what() << '\n';
         gc();
-        throw runtime_error("Error Setting up the Receiver");
+        throw RetryableError();
```

Comment:
It might be good to find the error case 1 level down in receiver.  Then you can rethrow e.